orion/database/db.go

```diff
@@ -1,66 +1,66 @@
 package database
 
 import (
-  "database/sql"
-  "fmt"
+	"database/sql"
+	"fmt"
 
-  "github.com/jmoiron/sqlx"
-  _ "github.com/go-sql-driver/mysql"
```

Comment:
we need to keep these `_`.
These underscores are like "partial" imports. They tell the program to import what it needs as opposed to the whole library.

Reply:
Does IntelliJ remove these?